src/server/dfly_main.cc

```diff
@@ -594,7 +595,20 @@ void sigill_hdlr(int signo) {
   exit(1);
 }
 
+void print_basic_usage_info() {
+  std::cout << "* Default logdir is: \"/tmp/\". It can be overwritten with flag --log_dir=[path]\n";
+  std::cout << "* Logfile name format is: ";
+  std::cout << "dragonfly.[machine_name].[user_name].[WARNING|INFO|ERROR].[date and time]\n";
```

Comment:
> but maybe we can at least provide the default here? 
Yes the default value for logdir is `""` which maps to `"/tmp"`. For non default values we can just print the flag value. However for the later, I wanted to provide a simple base line, because as we extend this and add more info in the future, this won't necessary be the most readable text as it will clutter. So maybe we should think/design something that is extendable and user-friendly. 

P.s. maybe I missed something in your comment here -- let me know if I did :)

Reply:
Looking a bit more into the glog API, maybe we could print something like:
```cpp
std::cout << "Logs will be written to the following paths (in this order): \n"
for (const auto& dir : GetLoggingDirectories()) {
  std::cout << dir << "/dragonfly.*\n";
}
```

That would be the most accurate, but also confusing. We could also "gamble" and only print the first one, but perhaps that's not going to be the one chosen (like if the directory doesn't exist or we lack write permissions, etc)

Reply:
This a little `cumbersome` and I ended up using `FLAGS_log_dir` . Let me know if you have any objections :)